profile/Readme.md

```diff
@@ -5,55 +5,26 @@
-    <img src="https://img.shields.io/github/downloads/TKMM-Team/Tkmm/total?style=flat&labelColor=2A2C33&logoColor=008000&color=008000&logo=github" alt="Downloads"/>
-  </a>
+  <a href="https://tkmm.org/downloads">
+    <img alt="Downloads" src="https://img.shields.io/badge/Windows%2C%20MacOS%2C%20Linux-%233ea45?style=flat&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjBweCIgdmlld0JveD0iMCAtOTYwIDk2MCA5NjAiIHdpZHRoPSIyMHB4IiBmaWxsPSIjMDAwMDAwIj48cGF0aCBkPSJNMjE2LTE0NHEtMjkuNyAwLTUwLjg1LTIxLjE1UTE0NC0xODYuMyAxNDQtMjE2di01MjhxMC0yOS43IDIxLjE1LTUwLjg1UTE4Ni4zLTgxNiAyMTYtODE2aDIyOHExNS4zIDAgMjUuNjUgMTAuMjlRNDgwLTc5NS40MiA0ODAtNzgwLjIxdC0xMC4zNSAyNS43MVE0NTkuMy03NDQgNDQ0LTc0NEgyMTZ2NTI4aDUyOHYtMjI4cTAtMTUuMyAxMC4yOS0yNS42NVE3NjQuNTgtNDgwIDc3OS43OS00ODB0MjUuNzEgMTAuMzVRODE2LTQ1OS4zIDgxNi00NDR2MjI4cTAgMjkuNy0yMS4xNSA1MC44NVE3NzMuNy0xNDQgNzQ0LTE0NEgyMTZabTUyOC01NDlMNDEyLTM2MXEtMTEgMTEtMjUgMTAuNVQzNjItMzYycS0xMS0xMS0xMS0yNS41dDExLTI1LjVsMzMxLTMzMWgtODFxLTE1LjMgMC0yNS42NS0xMC4yOVE1NzYtNzY0LjU4IDU3Ni03NzkuNzl0MTAuMzUtMjUuNzFRNTk2LjctODE2IDYxMi04MTZoMTY4cTE1LjMgMCAyNS42NSAxMC4zNVE4MTYtNzk1LjMgODE2LTc4MHYxNjhxMCAxNS4zLTEwLjI5IDI1LjY1UTc5NS40Mi01NzYgNzgwLjIxLTU3NnQtMjUuNzEtMTAuMzVRNzQ0LTU5Ni43IDc0NC02MTJ2LTgxWiIvPjwvc3ZnPg%3D%3D&label=downloads&labelColor=%23FFFFFF">
+    <a href="https://tkmm.org/docs">
+    <img alt="Documentation" src="https://img.shields.io/badge/Documentation-%23FFFFFF?style=flat&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjBweCIgdmlld0JveD0iMCAtOTYwIDk2MCA5NjAiIHdpZHRoPSIyMHB4IiBmaWxsPSIjMDAwMDAwIj48cGF0aCBkPSJNMjE2LTE0NHEtMjkuNyAwLTUwLjg1LTIxLjE1UTE0NC0xODYuMyAxNDQtMjE2di01MjhxMC0yOS43IDIxLjE1LTUwLjg1UTE4Ni4zLTgxNiAyMTYtODE2aDIyOHExNS4zIDAgMjUuNjUgMTAuMjlRNDgwLTc5NS40MiA0ODAtNzgwLjIxdC0xMC4zNSAyNS43MVE0NTkuMy03NDQgNDQ0LTc0NEgyMTZ2NTI4aDUyOHYtMjI4cTAtMTUuMyAxMC4yOS0yNS42NVE3NjQuNTgtNDgwIDc3OS43OS00ODB0MjUuNzEgMTAuMzVRODE2LTQ1OS4zIDgxNi00NDR2MjI4cTAgMjkuNy0yMS4xNSA1MC44NVE3NzMuNy0xNDQgNzQ0LTE0NEgyMTZabTUyOC01NDlMNDEyLTM2MXEtMTEgMTEtMjUgMTAuNVQzNjItMzYycS0xMS0xMS0xMS0yNS41dDExLTI1LjVsMzMxLTMzMWgtODFxLTE1LjMgMC0yNS42NS0xMC4yOVE1NzYtNzY0LjU4IDU3Ni03NzkuNzl0MTAuMzUtMjUuNzFRNTk2LjctODE2IDYxMi04MTZoMTY4cTE1LjMgMCAyNS42NSAxMC4zNVE4MTYtNzk1LjMgODE2LTc4MHYxNjhxMCAxNS4zLTEwLjI5IDI1LjY1UTc5NS40Mi01NzYgNzgwLjIxLTU3NnQtMjUuNzEtMTAuMzVRNzQ0LTU5Ni43IDc0NC02MTJ2LTgxWiIvPjwvc3ZnPg%3D%3D">
 </p>
 
-# Download TKMM
-
-* [Windows (x64/arm64)](https://tkmm-team.github.io/Tkmm-Wiki/downloads/index.html?tabs=windows)
-* [Linux (x64/arm64)](https://tkmm-team.github.io/Tkmm-Wiki/downloads/index.html?tabs=linux)
-* [macOS (x64/Apple Silicon)](https://tkmm-team.github.io/Tkmm-Wiki/downloads/index.html?tabs=macos)
-
-# Help & Documentation
-
-[TKMM's Support Website](https://tkmm-team.github.io/Tkmm-Wiki/index.html)
-
-# Discord
-
-[Join our Discord community](https://discord.com/invite/3tNUBWQRXv)
```

Comment:
It would have been better to just update the links with the new URL rather than delete that completely.

Reply:
This is supposed to be deduplication: It is located here and in the badge at the top of the file, and they lead to the same place. Though, they do serve different visual information so keeping both of them might be good.

Reply:
Put it in both places as often time text is easier to parse than badges.